### PR TITLE
Flow versions of Arcade SDK dependencies: XliffTasks, SourceLink, Pdb2Pdb

### PR DESCRIPTION
**Pull request by tmat**
## Description

Port of #4580, #3468, #5275 to 3.x

## Customer Impact

Allows us to automatically flow Arcade SDK dependencies.

## Regression

No

## Risk

Small.

## Workarounds

None.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -67,9 +67,13 @@
     <MicrosoftDotNetGitHubIssueLabelerAssetsVersion>1.4.0</MicrosoftDotNetGitHubIssueLabelerAssetsVersion>
     <MicrosoftMLVersion>1.0.0</MicrosoftMLVersion>
     <MicrosoftVisualStudioWebCodeGenerationDesignVersion>2.0.4</MicrosoftVisualStudioWebCodeGenerationDesignVersion>
-    <MicrosoftDiaSymReaderConverterVersion>1.1.0-beta1-62810-01</MicrosoftDiaSymReaderConverterVersion>
+    <MicrosoftDiaSymReaderConverterVersion>1.1.0-beta2-19521-03</MicrosoftDiaSymReaderConverterVersion>
+    <MicrosoftDiaSymReaderPdb2PdbVersion>1.1.0-beta2-19521-03</MicrosoftDiaSymReaderPdb2PdbVersion>
     <MicrosoftDiaSymReaderNativeVersion>1.7.0</MicrosoftDiaSymReaderNativeVersion>
     <MicrosoftDotNetMaestroClientVersion>1.1.0-beta.19454.3</MicrosoftDotNetMaestroClientVersion>
     <MicrosoftDotNetSwaggerGeneratorMSBuildVersion>1.0.0-beta.19459.10</MicrosoftDotNetSwaggerGeneratorMSBuildVersion>
+    <MicrosoftSourceLinkVersion>1.0.0-beta2-19554-01</MicrosoftSourceLinkVersion>
```
Comment (JohnTortugo) [inline eng/Versions.props:75]: Duplicated with line 77?

Comment (tmat) [reply eng/Versions.props]: Yup, thanks. Removed.

**Comment by tmat** [comment]
@JohnTortugo Also ported #5275 to get the latest Source Link build.

**Comment by tmat** [comment]
This needs to be done in 2 steps, first: https://github.com/dotnet/arcade/pull/5290

**Pull request merged**